src/bin/pgcopydb/catalog.c

```diff
@@ -6487,6 +6487,66 @@ catalog_add_s_extension_config(DatabaseCatalog *catalog,
 }
 
 
+/*
+ *   catalog_iter_s_extesnion_timescaledb_checker , iterates over the list of extensions in our catalogs
+ *   and checks for presence of timescaledb extension.
+ */
+bool
+catalog_iter_s_extension_timescaledb_checker(DatabaseCatalog *catalog,
+											 bool *timescaledb)
+{
```

Comment:
That looks like it should be a lookup function, not a iterator function that short-circuit the iteration when it finds a specific value?

Reply:
How about calling this as `catalog_has_timescaledb_extension`?

Reply:
> How about calling this as `catalog_has_timescaledb_extension`?
I’ve updated the name to catalog_has_timescaledb_extension as suggested.